src/GrpcProtos/capture.proto

```diff
@@ -677,6 +677,12 @@ message ErrorEnablingUserSpaceInstrumentationEvent {
   bytes message = 2;
 }
 
+message InfoEnablingUserSpaceInstrumentationEvent {
```

Comment:
The first question that comes to mind is if we need a new message or if this already fits in `ErrorEnablingUserSpaceInstrumentationEvent`.

Reply:
You are right, I also don't really have an opinion.

But: I think we should find a slightly better name for the event, as it's not generic "info" but a specific error/warning, i.e., a few functions couldn't be instrumented. I can think of "ErrorInstrumentingFunctionsWithUserSpaceInstrumentationEvent", but it's incredibly long, so we can do better, but in general I would like the name to be more specific.

Reply:
I don't like the "Error" in that name. I guess "Warning" is the better wording. 

WarningInstrumentingWithUserSpaceInstrumentationEvent it is.